### PR TITLE
chore(tools): update typescript

### PR DESCRIPTION
**Pull request by robwormald**
closes https://github.com/angular/angular/issues/4653

### CODE REVIEW

File: tools/typescript.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-//typescript v 1.6.0-beta
```
Comment (IgorMinar) [inline tools/typescript.js]: can you keep + update the version number? otherwise we won't know what is being used.

**Comment by IgorMinar** [comment]
I assume that you have commit rights, so go ahead and merge this in.

**Comment by robwormald** [comment]
added version number, i don't have commit rights on this one.

**Comment by IgorMinar** [comment]
landed as 6c9d0b11b5bde440e06e5f0c3fa2c1b9c8b3a0af

**Pull request closed without merging**